src/DotLiquid.Tests/Tags/CommentTests.cs

```diff
@@ -0,0 +1,45 @@
+using System;
+using System.Collections.Generic;
+using DotLiquid.Exceptions;
+using NUnit.Framework;
+using DotLiquid.Tags;
+
+namespace DotLiquid.Tests.Tags
+{
+    [TestFixture]
+    public class CommentTests
+    {
+        [Test]
+        public void TestEmptyComment()
+        {
+            Assert.AreEqual(string.Empty, Template.Parse("{% comment %}{% endcomment %}").Render());
+            // Next test is specific to legacy parser and was removed from Ruby Liquid. Test that it is ignored is in TestShortHandSyntaxIsIgnored
```

Comment:
nit nit: have an empty line before an in-line code comment line
```suggestion

            // Next test is specific to legacy parser and was removed from Ruby Liquid. Test that it is ignored is in TestShortHandSyntaxIsIgnored
```